Orange/tests/test_classification.py

```diff
@@ -77,10 +77,10 @@ def test_value_from_probs(self):
         clf = learn(t)
         clf.ret = Model.Probs
         y2 = clf(x, ret=Model.Value)
-        self.assertTrue(y2.shape == (nrows,))
+        self.assertEqual(y2.shape, (nrows,))
         y2, probs = clf(x, ret=Model.ValueProbs)
-        self.assertTrue(y2.shape == (nrows, ))
-        self.assertTrue(probs.shape == (nrows, 2))
+        self.assertEqual(y2.shape, (nrows, ))
```

Comment:
This bad-whitespace seems to be one of the [21 errors you introduced](https://travis-ci.org/biolab/orange3/jobs/119871036).